src/neuroconv/datainterfaces/icephys/baseicephysinterface.py

```diff
@@ -33,6 +26,17 @@ def get_source_schema(cls) -> dict:
         return source_schema
 
     def __init__(self, file_paths: list):
+        # Check if the ndx_dandi_icephys module is available
+        dandi_icephys_spec = importlib.util.find_spec("ndx_dandi_icephys")
+        if dandi_icephys_spec is not None:
+            dandi_icephys_module = importlib.util.module_from_spec(dandi_icephys_spec)
+            dandi_icephys_spec.loader.exec_module(dandi_icephys_module)
+            self.DandiIcephysMetadata = getattr(dandi_icephys_module, "DandiIcephysMetadata")
```

Comment:
```suggestion
            from ndx_dandi_icephys import DandiIcephysMetadata
```

I mean just to determine if the package is installed; classic import syntax is cleaner than the rest once you know it's safe to call

Reply:
Sure.